deployment/v3/docs/change-domain-name-in-a-running-env.md

```diff
@@ -0,0 +1,96 @@
+# Change domain name in a running env ( v2 )
+
+Follow the below steps to rename domain name in a running v2 env:
+1. Update the domain name in groups/all.yml
+2. Ran playbooks/nginx.yml
+3. Redeploy config-server, keycloak
+4. Update  the domain name in the  valid redirect URL & web origins of mosip keycloak-clients 
```

Comment:
Update the domain name in the valid redirect URL & web origins of MOSIP keycloak-clients.

Remove extra spaces